src/theory/quantifiers/sygus_sampler.cpp

```diff
@@ -561,7 +561,7 @@ Node SygusSampler::getRandomValue(TypeNode tn)
       {
         d_rstring_alphabet.push_back(ch);
         Trace("sygus-sample-str-alpha")
-            << " \"" << String::convertUnsignedIntToChar(ch) << "\"";
+            << " \"" << static_cast<char>(ch) << "\"";
```

Comment:
Not a huge deal but this cast will potentially lose some bits.

Reply:
Ill change to the \uN format.